lib/gcpspanner/searchtypes/features_search_visitor.go

```diff
@@ -50,6 +50,22 @@ func (v *FeaturesSearchVisitor) addError(err error) {
 	v.err = errors.Join(v.err, err)
 }
 
+type termMissingValueError struct {
+	term SearchIdentifier
+}
+
+func (e termMissingValueError) Error() string {
+	return fmt.Sprintf("term %s is missing value", e.term)
+}
+
+type termMissingRangeValueError struct {
+	term SearchIdentifier
+}
+
+func (e termMissingRangeValueError) Error() string {
+	return fmt.Sprintf("term %s is missing value", e.term)
```

Comment:
Should this message be different?  E.g., value -> value range.

Reply:
Good catch! Will adjust